database/migrations/2017_04_04_202103_UpdateSharedExpensesDateSettled.php

```diff
@@ -0,0 +1,36 @@
+<?php
```

Comment:
This file is not needed anymore as Vince also made changes to the tables in his previous branch